channels/email/email-headers.rst

```diff
@@ -14,35 +15,25 @@ Header checks are case insensitive.
 
 .. tip::
 
-   Below headers are examples and –in our opinion– the most relevant ones.
-   However: You can adjust mostly any article or ticket attribute (yes, custom
-   ones as well) if you know the attribute's exact name.
-
-   The name column within :doc:`object's management </system/objects>` provides
-   easy access to objects attribute names. 🤓
-
-Trigger auto responses
-----------------------
+   - Below headers are examples and in our opinion the most relevant ones.
```

Comment:
"Below headers" / "Below options" etc. - i think this does not work in English. "The options (described) below" may work, not sure. @dvuckovic?

Reply:
Several occurrences, please check the entire file, or even the whole doc in another MR if you want.

Reply:
`Headers listed below...`

or alternatively:

`Header names listed below...`